gemd/demo/cake.py

```diff
@@ -18,6 +18,7 @@
 from gemd.entity.util import complete_material_history, make_instance
 from gemd.entity.file_link import FileLink
 from gemd.entity.source.performed_source import PerformedSource
+from gemd.json import GEMDJson
```

Comment:
Is this a bugfix? I was expecting a replace not add.

Reply:
I had accidentally removed it from the copy in an earlier commit.  You can see it in the original `cake.py` [here](https://github.com/CitrineInformatics/taurus/blob/f49a2b27fe4a27f30903aa924a18b347de630858/taurus/demo/cake.py#L22)